azkaban-exec-server/src/test/java/azkaban/execapp/JobRunnerTest.java

```diff
@@ -298,20 +302,19 @@ public void testDelayedExecutionCancelledJob() throws InterruptedException {
     Assert.assertTrue("Node status is " + node.getStatus(),
         node.getStatus() == Status.KILLED);
     Assert.assertTrue(node.getStartTime() > 0 && node.getEndTime() > 0);
-    Assert.assertTrue(node.getEndTime() - node.getStartTime() < 1000);
-    Assert.assertTrue(node.getStartTime() - startTime >= 1000);
-    Assert.assertTrue(node.getStartTime() - startTime <= 4000);
+    Assert.assertTrue(node.getEndTime() - node.getStartTime() < 2000);
```

Comment:
After increasing the sleep time to 2s, I can understand that (node.getStartTime - startTime) is changed, but why is (node.getEndTime - node.getStartTime) also affected?

Reply:
Honestly I haven't given it any deeper thought. I'm merely changing it back to [how it used to be](https://github.com/azkaban/azkaban/commit/eb6bdd5e4bb25e1effcbe5333fc7768ec79b7274#diff-e0cbe4296eb29a4206100f4f2fbc2c28). Now that we have the Awaitility in place, having 2s instead of 1s is afforable.

Reply:
I agree with changing 1s to 2s in the sleep time. But the check in line 305 seems to be independent of this change. The job is configured to run for 1 sec in `createJobRunner(1, "testJob", 1, false, loader, eventCollector);`, so I think the original intention was to check whether the job finished within 1 sec. Also it seems it used to be `Assert.assertTrue(node.getEndTime() - node.getStartTime() < 1000);` from the link you shared?

Reply:
Oh sorry, I can see I made a mistake there. Changing back to 1000.